text/trace/0107-sampling-score.md

```diff
@@ -0,0 +1,217 @@
+# Associating sampling score with the trace
+
+Enable consistent sampling across distributed application with different
+sampling rates and probability calculation algorithms.
+
+## TL;DR
+
+**Score** is a floating point number associated with
+the trace. It's calculated when trace starts and flows in the `tracestate`,
+it's used by samplers to make consistent sampling decisions.
```

Comment:
May want to clarify that this is not the initial sampling probability, but a score that combined with the local probability determines if a trace is sampled or not.